### PR TITLE
Update font name IDs to include unicode characters, update URLs

### PR DESCRIPTION
**Pull request by arrowtype**
Based on feedback from an anonymous expert, we have become aware that a few changes are useful in the `name` table:

- Name 4 should match Name 1 in a single-style font
- Names 8, 9, and 10 can include Chinese characters, to allow the full Latin & Chinese name `Lisa Huang 黄丽莎`

This also bumps the version number to 1.002. It is now (incorrectly) set at 1.000 in the name table.

Something I am uncertain about: names 1 & 4 can contain unicode characters, and so should potentially be `Noto Sans Nüshu` (with a `ü`). However, Dave said something ambiguous that seemed to possibly contradict this. So, for now, this matches them. I am happy to also add the `ü` if I get confirmation that we can.

The `name` table now has these diffs (determined via [fdiff](https://github.com/source-foundry/fdiff)):

```diff
@@ -12,13 +12,13 @@
<name>
    <namerecord nameID="0" platformID="3" platEncID="1" langID="0x409">
      Copyright 2020 Google Inc. All Rights Reserved.
    </namerecord>
    <namerecord nameID="1" platformID="3" platEncID="1" langID="0x409">
      Noto Sans Nushu
    </namerecord>
    <namerecord nameID="2" platformID="3" platEncID="1" langID="0x409">
      Regular
    </namerecord>
     <namerecord nameID="3" platformID="3" platEncID="1" langID="0x409">
-      1.000;GOOG;NotoSansNushu-Regular
+      1.002;GOOG;NotoSansNushu-Regular
     </namerecord>
     <namerecord nameID="4" platformID="3" platEncID="1" langID="0x409">
-      Noto Sans Nushu Regular
+      Noto Sans Nushu
     </namerecord>
     <namerecord nameID="5" platformID="3" platEncID="1" langID="0x409">
-      Version 1.000
+      Version 1.002
     </namerecord>
     <namerecord nameID="6" platformID="3" platEncID="1" langID="0x409">
       NotoSansNushu-Regular
@@ -27,25 +27,25 @@
       Noto is a trademark of Google Inc.
     </namerecord>
     <namerecord nameID="8" platformID="3" platEncID="1" langID="0x409">
-      Lisa Huang
+      Lisa Huang 黄丽莎
     </namerecord>
     <namerecord nameID="9" platformID="3" platEncID="1" langID="0x409">
-      Lisa Huang
+      Lisa Huang 黄丽莎
     </namerecord>
     <namerecord nameID="10" platformID="3" platEncID="1" langID="0x409">
-      Designed by Lisa Huang
+      Designed and produced by Lisa Huang 黄丽莎
     </namerecord>
     <namerecord nameID="11" platformID="3" platEncID="1" langID="0x409">
-      http://www.google.com/get/noto/
+      https://github.com/notofonts/noto-sans-nushu
     </namerecord>
     <namerecord nameID="12" platformID="3" platEncID="1" langID="0x409">
-      http://www.lisahuang.work
+      https://www.lisahuang.work/
     </namerecord>
     <namerecord nameID="13" platformID="3" platEncID="1" langID="0x409">
       This Font Software is licensed under the SIL Open Font License, Version 1.1. This Font Software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the SIL Open Font License for the specific language, permissions and limitations governing your use of this Font Software.
     </namerecord>
     <namerecord nameID="14" platformID="3" platEncID="1" langID="0x409">
-      http://scripts.sil.org/OFL
+      https://scripts.sil.org/OFL
     </namerecord>
   </name>
```

This PR also includes a fix to a Unicode mapping error, details in a comment below.

### CODE REVIEW

**Comment by marekjez86** [comment]
Looks good to me... re Nüshu -- let's try and test in some of the apps and systems

**Comment by davelab6** [comment]
I agree that testing would be good,. @rsheeter does the API support non ASCII family names?

**Comment by arrowtype** [comment]
Update on the non-name diffs, after consulting @LisaHuang2017:

The font had correct unicodes at bfd1122605006f29a2072952baa66b4880bd9b76, but then a misunderstanding of a Unicode update prompted an incorrect mapping update in 21ad41c3c15e2cd01b398f1469d3b6e90cab483f. Right now, the current release ([1.001](https://github.com/notofonts/noto-sans-nushu/releases/tag/1.001)) has the incorrect mapping, even though the [current latest TTF in the font directory](https://github.com/notofonts/noto-sans-nushu/tree/c89f05023df1d49707dd16f8e7a3d01e498a37b8/fonts) does have the correct mapping.

TL;DR: this PR has the correct Unicodes, and we should update the Releases once it is merged.

Below is an image of the diffs; left is incorrect, right is in this PR.

image

**Comment by arrowtype** [comment]
Okay, I think that names 1 & 4 should be `Nushu`, because:

1. No other Noto font names include accented characters.
2. It would be ambiguous to users how to make API calls. 

I’m converting this PR from draft to “Ready for review.” 

@marekjez86 or @davelab6, please merge it unless you wish to request any changes.
